src/dmd/dimport.d

```diff
@@ -145,8 +147,7 @@ extern (C++) final class Import : Dsymbol
                 return true;
             }
         }
-        Dsymbol s = dst.lookup(id);
-        if (s)
+        if (Dsymbol s = dst.lookup(id))
```

Comment:
please do not mix refactorings with feature implementation

Reply:
Done.